some_string: "cat is barking"
